File: exercises/beecrowd/1002/1002.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var raio float64
	pi := 3.14159

	fmt.Scanln(&raio)

	result := math.Round((pi*(raio*raio))*10000) / 10000

	fmt.Printf("A=%.4f\n", result)
}

File: exercises/beecrowd/1002/1002.ts
// https://judge.beecrowd.com/pt/problems/view/1002
import * as fs from 'fs';

function main(): void {
    const input = fs.readFileSync('/dev/stdin', 'utf8');
    const lines = input.split(/\r?\n/);

    const PI = 3.14159
    const raio = Number(lines.shift());

    const result = (Math.round((PI*(raio*raio))*10000) / 10000).toFixed(4)

    console.log(`A=${result}`);
}

main()